docs/changelog/changelog_en.md

```diff
@@ -1,49 +1,46 @@
-- Headlines now use the configured font.
-- Fixed layout shift on category page.
-- The build script failed on Windows because of incompatibilities between file name pattern and operating system. The file name pattern now works on Windows.
+- Fixed an issue where increasing the quantity beyond maximum stock would clear the cart.
+- Removed the "Add to Cart" notification from item and category pages when the quick checkout modal is not present.
+- Improved accessibility for the cart and wishlist pages by increasing the font size.
+- Fixed an issue where the product path was not reactive when the category was changed.
+- The build now automatically generates a language file for every active language, not just the default language.
+- Resolved an issue where the soft login was still shown after successful authentication on the order confirmation page.
+- Corrected manufacturer data to use `externalName` instead of `name` in structured data.
+- Updated headlines to use the configured font.
+- Fixed layout shift on the category page.
+- Resolved build script failure on Windows due to file name pattern incompatibilities.
 - The build script now adds the `API_URL` to the environment if it exists.
 
 ### 👷 Changed
 
-- The `height` and `width` attributes that where set in terra ui are used only for full size images of an item.
-- Generating missing language files based on the language configuration from the environment has been moved to the build script. The locale configuration is now based on the language files in the `lang` directory.
-- Started to unify the SDK/API error handling. Errors do now return keys that can be translated in the frontend.
-- Manufacturer visual improvments
-- Changed manufacturer translation text.
-- The payment status on an order now supports a second argument, allowing you to define custom translations for the different payment states.
-- In cases where the basket is empty during the checkout process, the system will now redirect to `/cart` and display a notification.
+- Implemented a new notification design.
 - Increased default notification timeout from 3 to 5 seconds.
+- The `height` and `width` attributes set in Terra UI are now used only for full-size images of an item.
+- Moved the generation of missing language files based on the environment's language configuration to the build script. The locale configuration is now based on the language files in the `lang` directory.
+- Started unifying SDK/API error handling. Errors now return keys that can be translated in the frontend.
+- Improved manufacturer visuals.
+- Updated manufacturer translation text.
+- Added support for a second argument to the payment status on an order. This allows you to define custom translations for different payment states.
+- If the cart is empty during the checkout process, the user is now redirected to the cart.
+- The logo container is now more flexible and adapts to the dimensions of the provided logo.
```

Comment:
```suggestion
- The logo container is now more flexible and adapts to the dimensions of the provided custom logos.
```

Reply:
I don't think "custom" adds any valuable information here. In the end all logos are custom logos, even the plenty one we use.